spec/queries/articles/api_search_query_spec.rb

```diff
@@ -22,6 +22,13 @@
       articles = described_class.call({ q: "ruby" })
       expect(articles.count).to eq(1)
     end
+
+    it "does not show article if below minimum index" do
```

Comment:
I assume the other tests implicitly give us a `it "does show article if gte minimum index"` test then?